app/api/schema/events.py

```diff
@@ -283,6 +283,13 @@ def validate_timezone(self, data, original_data):
                                 schema='CustomFormSchema',
                                 many=True,
                                 type_='custom-form')
+    owner = Relationship(attribute='owner',
+                         self_view='v1.event_owner',
+                         self_view_kwargs={'id': '<id>'},
+                         related_view='v1.user_list',
```

Comment:
how can an event have multiple owners, wasn't the whole point of the owner was to differentiate between creator and organizers?

Reply:
@CosmicCoder96 missed it. Fixed now